src/index.js

```diff
@@ -9,12 +9,15 @@
  */
 import Layer from './api/layer';
 import Dataset from './api/dataset';
+import * as expressions from './core/style/functions';
 import { Style, parseStyle } from './core/style';
 
 // Source namespace has dataset and SQL
 const source = { Dataset };
 
+
 export { Layer };
 export { source };
 export { Style };
 export { parseStyle }; // TODO(@iago): temporal export, we must find a better place
+export { expressions };
```

Comment:
we want to export this as `carto.expressions` or we want `carto.style.<expression>` ? @davidmanzanares @Jesus89

Reply:
I vote for `carto.style.expression` but we can talk about this in the `Expressions` epic.